procedures/katello/branch.md.erb

```diff
@@ -90,6 +90,12 @@
       cp -rf public/apipie-cache/* $GITDIR/theforeman.org/plugins/katello/$VERSION/api
       find $GITDIR/theforeman.org/plugins/katello/$VERSION/api -name "*.json" -type f -delete
       sed -i "/layout: /d" $GITDIR/theforeman.org/plugins/katello/$VERSION/api/index.md
+- [ ] Prepare "Katello Next" and future redmine versions
+  - [ ] Rename the "Katello Next" release to be the newest Katello x.y.0 release (e.g. Katello 4.5.0)
```

Comment:
We have a variable for this:
```suggestion
  - [ ] Rename the "Katello Next" release to be the newest Katello <%= develop %>.0 release
```

It's defined here:
https://github.com/theforeman/tool_belt/blob/00e5cc21bad5c4db06acd4f1066e01373cd7fa77/lib/tool_belt/commands/procedure.rb#L21